### ISSUE TITLE
What's the accuracy of deit-S without pre-trained on CIFAR10

### ISSUE THREAD

**Issue by hanwenran1** [facebookresearch/deit#202]
Hi,

What's the accuracy of deit-S without pre-trained on CIFAR10? Mine is only 63.2. Would you like to told me yours? And my hyper-parameters follow this link[https://github.com/facebookresearch/deit/issues/45#issuecomment-765213622](url)

Thanks.

**Comment by TouvronHugo** [comment]
Hi @hanwenran1 ,
It's 98.5%.
Best regards,
Hugo
